apis/python/src/tiledbsoma/options/_soma_tiledb_context.py

```diff
@@ -47,7 +48,9 @@ def _maybe_timestamp_ms(input: Optional[OpenTimestamp]) -> Optional[int]:
 """Sentinel object to distinguish default parameters from None."""
 
 
-class SOMATileDBContext:
+class SOMATileDBContext(clib.SOMAContext):  # type: ignore
+    # (Class cannot subclass "SOMAContext" (has type "Any"))
```

Comment:
This one too. Nothing that I found via Google on my own resolved it.

Reply:
This one is, thankfully, a little more straightforward. It’s because [the type checker has no information about what a `clib.SOMAContext`](https://stackoverflow.com/questions/49888155/class-cannot-subclass-qobject-has-type-any-using-mypy) is, so it treats it as an `Any` type in some ways. mypy disallows this by default, since that means that `SOMATileDBContext` would effectively be an `Any` as far as accessing it is concerned. So if you had `my_context.this_member_will_never_exist`, it would always be forced to say "yeah that maybe exists" and call the result of that call an `Any`.

I would handle this by using composition rather than inheritance, so maybe have a `SOMATileDBContext._native_context` field that contains the `clib.SOMAContext`.